updater/README.md

```diff
@@ -2,4 +2,50 @@
 
 The updater is a separate binary that runs as a separate process to update collector artifacts (including the collector itself) when managed by [BindPlane OP](https://github.com/observIQ/bindplane-op).
 
-Because the updater edits service configurations, it needs elevated privileges to run (root on Linux + macOS, administrative privileges on Windows).
+Because the updater interacts with the service manager, and may edit privileged files, it needs elevated privileges to run (root on Linux + macOS, administrative privileges on Windows).
+
+## Updating Overview
+
+1. The collector receives PackagesAvailable message.
+2. The collector downloads tarball containing new updater and updated artifacts (e.g. new collector, plugins, etc.) based on the contents of the PackagesAvailable message.
+3. The collector unpacks tarball into `$INSTALL_DIR/tmp/latest`.
+4. The collector copies the newest updater binary from `$INSTALL_DIR/tmp/latest` to the working directory.
+5. The collector starts the updater in as a separate process in a new process group.
+   * If the updater fails to stop the collector within 30 seconds, the collector will kill the updater and abort the update. 
+
+6. The updater starts, then shuts down the collector through the service manager.
+7. The collector shuts down, orphaning the updater process.
```

Comment:
minor, but step 7 sounds partly like it's repeating part 6. maybe "The updater process is orphaned when the collector shuts down". either that or combine them both into one step.